pages/manager/view-employee-barrier.tsx

```diff
@@ -19,6 +19,13 @@ const ReviewBarrier: React.FunctionComponent = () => {
         defaultMessage:
           "Frank's Barrier: Noise in the workplace - GC Workplace Accessibility Passport",
       })}
+      crumbs={[
+        { title: "Frank Turot", href: "/manager/view-employee-passport" },
+        {
+          title: "Noise in the workplace",
+          href: "/manager/view-employee-barrier",
```

Comment:
The `href` should be removed from the last elemen, since the user is on this page. Can you do it for all the other pages as well.